ochre/utils/schedule.py

```diff
@@ -16,59 +16,58 @@
-        'clothes_washer': 'Clothes Washer',
-        'clothes_dryer': 'Clothes Dryer',
-        'dishwasher': 'Dishwasher',
-        'refrigerator': 'Refrigerator',
-        'cooking_range': 'Cooking Range',
-        'lighting_interior': 'Indoor Lighting',
-        'lighting_exterior': 'Exterior Lighting',
-        'lighting_basement': 'Basement Lighting',
-        'lighting_garage': 'Garage Lighting',
-        'plug_loads_other': 'MELs',
-        'plug_loads_tv': 'TV',
-        'plug_loads_well_pump': 'Well Pump',
+    "Power": {
+        "clothes_washer": "Clothes Washer",
+        "clothes_dryer": "Clothes Dryer",
+        "dishwasher": "Dishwasher",
+        "refrigerator": "Refrigerator",
+        "cooking_range": "Cooking Range",
+        "lighting_interior": "Indoor Lighting",
+        "lighting_exterior": "Exterior Lighting",
+        "lighting_basement": "Basement Lighting",
+        "lighting_garage": "Garage Lighting",
+        "plug_loads_other": "MELs",
+        "plug_loads_tv": "TV",
+        "plug_loads_well_pump": "Well Pump",
         # 'plug_loads_vehicle': 'electric vehicle charging',  # Not using scheduled EV load
-        'fuel_loads_grill': 'Gas Grill',
-        'fuel_loads_fireplace': 'Gas Fireplace',
-        'fuel_loads_lighting': 'Gas Lighting',
-        'pool_pump': 'Pool Pump',
-        'pool_heater': 'Pool Heater',
-        'hot_tub_pump': 'Hot Tub Pump',
-        'hot_tub_heater': 'Hot Tub Heater',
-        'ceiling_fan': 'Ceiling Fan',
+        "fuel_loads_grill": "Gas Grill",
+        "fuel_loads_fireplace": "Gas Fireplace",
+        "fuel_loads_lighting": "Gas Lighting",
+        "pool_pump": "Pool Pump",
+        "pool_heater": "Pool Heater",
+        "permanent_spa_pump": "Spa Pump",
+        "permanent_spa_heater": "Spa Heater",
+        "ceiling_fan": "Ceiling Fan",
         # 'vent_fan': 'Ventilation Fan',  # not included in schedule
         # 'basement_mels': 'Basement MELs',  # not modeled
     },
-    'Water': {
-        'hot_water_fixtures': 'Water Heating',
-        'hot_water_clothes_washer': 'Clothes Washer',
-        'hot_water_dishwasher': 'Dishwasher',
+    "Water": {
+        "hot_water_fixtures": "Water Heating",
+        "hot_water_clothes_washer": "Clothes Washer",
+        "hot_water_dishwasher": "Dishwasher",
+    },
+    "Setpoint": {
+        "heating_setpoint": "HVAC Heating",
+        "cooling_setpoint": "HVAC Cooling",
+        "water_heater_setpoint": "Water Heating",
     },
-    'Setpoint': {
-        'heating_setpoint': 'HVAC Heating',
-        'cooling_setpoint': 'HVAC Cooling',
-        'water_heater_setpoint': 'Water Heating',
+    "Ignore": {
+        "extra_refrigerator": None,
+        "freezer": None,
+        "clothes_dryer_exhaust": None,
+        "lighting_exterior_holiday": None,
+        "plug_loads_vehicle": None,
+        "battery": None,
+        "vacancy": None,
+        "water_heater_operating_mode": None,
+        "Vacancy": None,
+        "Power Outage": None,
```

Comment:
I should maybe make an issue/TODO around reading outage values, if we don't already have one. It's something we've done in OCHRE before. It's niche in ResStock but we might want to be able to handle it at some point